pkg/network/ebpf/c/protocols/classification/usm-context.h

```diff
@@ -23,6 +23,7 @@ typedef struct {
     // bit mask with layers that should be skiped
     u16 routing_skip_layers;
     classification_prog_t routing_current_program;
+    __u8 tls_content_type;
```

Comment:
why is it here?
why can't it be part of the new structs you're introducing?

Reply:
we discussed offline but we are going to rename this struct to `classification_ctx` after this PR is merged. this field fits better here because it has information relevant to the current connections classification context, where the new map `tls_enhanced_tags` is specifically for the output tags from the classification. the struct in that map is embedded into `conn_t` so we'd like to keep it small

Reply:
I'm going to remove this entirely